.gitignore.orig

```diff
@@ -0,0 +1,86 @@
+# Prerequisites
+*.d
+
+# Compiled Object files
+*.slo
+*.lo
+*.o
+*.obj
+
+# Precompiled Headers
+*.gch
+*.pch
+
+# Compiled Dynamic libraries
+*.so
+*.dylib
+*.dll
+
+# Fortran module files
+*.mod
+*.smod
+
+# Compiled Static libraries
+*.lai
+*.la
+*.a
+*.lib
+
+# Executables
+*.exe
+*.out
+*.app
+
+# CMake
+CMakeFiles/*
+cmake-build-*
+CMakeCache.txt
+cmake_install.cmake
+src/test/CMakeFiles/
+src/test/cmake_install.cmake
+src/test/CMakeCache.txt
+src/test/Makefile
+Makefile
+
+# IDE
+.idea/
+
+# Dataset
+*.json
+
+# Clang format
+.clang-format
+
+# Cython
+python/quartz/_cython/*.cpp
+python/build/*
+python/dist/*
+python/quartz.egg-info/
+python/quartz/__pycache__/
+src/python/utils/__pycache__/
+src/python/verifier/__pycache__/
+install_manifest.txt
+**/__pycache__
+
+# Build
+build/*
+
+# others
+.DS_Store
+**/wandb
+
+experiment/pretrain/dataset
+experiment/pretrain/pretrain_dataset*
+experiment/pretrain/outputs
+
+<<<<<<< HEAD
+=======
+experiment/ppo/PPO_logs
+experiment/ppo/PPO_preTrained
+experiment/ppo/pretrain/dataset
+experiment/ppo/pretrain/outputs
+experiment/ppo/pretrain/pretrain_dataset_g9
+experiment/ppo/pretrain/pretrain_dataset_g99
+
+experiment/a2c/*txt
+>>>>>>> Added more files in .gitignore

```

Comment:
Please fix this... Should this file `.gitignore.orig` be removed or replace `.gitignore`?

Reply:
I will remove this file